visma/functions/variable.py

```diff
@@ -77,3 +76,184 @@ def integrate(self, wrtVar):
 
     def calculate(self, val):
         return self.coefficient * ((val**(self.power)))
+
+    def __radd__(self, other):
+        return self + other
+
+    def __add__(self, other):
+        from visma.functions.constant import Constant
+        if isinstance(other, Variable):
+            sortedValuesSelf = sorted(self.value)
+            sortedValuesOther = sorted(other.value)
+            if self.coefficient == 0:
+                return Constant(0, 1, 1)
+            if (self.power == other.power) & (sortedValuesSelf == sortedValuesOther):
+                if self.before == '-':
+                    self.coefficient -= other.coefficient
+                else:
+                    self.coefficient += other.coefficient
+                return self
+        elif isinstance(other, Constant):
+            expression = Expression()
+            expression.tokens = [self]
+            expression.tokens.extend([Plus(), other])
+            self = expression
+            return expression
+        elif isinstance(other, Expression):
+            from visma.functions.operator import Binary
```

Comment:
Already imported.
```suggestion
```
Whenever importing write the import line at the start of the file unless in the case of cyclic imports.